UI/slider-ignorewheel.cpp

```diff
@@ -20,3 +21,82 @@ void SliderIgnoreScroll::wheelEvent(QWheelEvent *event)
 	else
 		QSlider::wheelEvent(event);
 }
+
+VolumeSlider::VolumeSlider(obs_fader_t *fader, QWidget *parent)
+	: SliderIgnoreScroll(parent)
+{
+	fad = fader;
+}
+
+VolumeSlider::VolumeSlider(obs_fader_t *fader, Qt::Orientation orientation,
+			   QWidget *parent)
+	: SliderIgnoreScroll(orientation, parent)
+{
+	fad = fader;
+}
+
+VolumeAccessibleInterface::VolumeAccessibleInterface(QWidget *w)
+	: QAccessibleWidget(w)
+{
+}
+
+VolumeSlider *VolumeAccessibleInterface::slider() const
+{
+	return qobject_cast<VolumeSlider *>(object());
+}
+
+QString VolumeAccessibleInterface::text(QAccessible::Text t) const
```

Comment:
Shouldnt this also provide a name/desc for the widget? (In the QAccessible::Text::Name/Description cases)

Reply:
QAccessibleWidget has defaults that didn't need changing. The only thing I needed to provide my own alternate text for was the value. I would only have needed to provide my own text if I extended the much more bare QAccessibleInterface.

Reply:
It looked like it would override the default qaccessibilitywidget information, thanks for explaining.